lib/src/_transport.dart

```diff
@@ -63,8 +63,11 @@ class Response {
   Map<String, dynamic>? _bodyAsMap;
   List? _bodyAsList;
 
+  /// The HTTP headers of the response.
+  final Map<String, String> headers;
+
   /// Creates a new [Response] object.
-  Response(this.statusCode, this.body);
+  Response(this.statusCode, this.body, this.headers);
```

Comment:
This seems to be  breaking API change, which would need to bump the version. I don't mind that, but then please update it to 0.4.0.

Reply:
I bumped the version to 0.3.12